Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -12690,15 +12696,22 @@ - (void)contextMenuInteraction:(UIContextMenuInteraction *)interaction willEndFo
     _contextMenuHasRequestedLegacyData = NO;
     _contextMenuElementInfo = nullptr;
 
-    [animator addCompletion:[weakSelf = WeakObjCPtr<WKContentView>(self)] () {
+    auto lambda = [weakSelf = WeakObjCPtr<WKContentView>(self)] () {
```

Comment:
this should probably have a better name than just :lambda:?

Reply:
I could name it `tearDownAfterAnimation` but if animator is nil (like on Catalyst) it happens immediately. 

I'm certainly open to suggestions but previously the lambda had no name so using lambda seemed reasonable.

Reply:
contextFinalizer or something?